src/components/PrimaryPanes/index.js

```diff
@@ -38,13 +43,6 @@ type Props = {
 };
 
 class PrimaryPanes extends Component<Props, State> {
-  renderShortcut: Function;
-  selectedPane: string;
-  showPane: Function;
-  renderTabs: Function;
-  renderChildren: Function;
-  onAlphabetizeClick: Function;
```

Comment:
Many of these fields don't exist anymore or are unnecessary, so I removed them. Let me know if any of these were a mistake, though!

Reply:
thanks